lms/djangoapps/teams/views.py

```diff
@@ -1277,13 +1277,12 @@ class CreateTeams(GenericAPIView):
     def post(self, request, course_id):
 
         if not has_team_api_access(request.user, course_id) :
-            return Response(status=status.HTTP_404_NOT_FOUND)
+            return Response(status=status.HTTP_400_BAD_REQUEST)
 
         file = request.FILES
         if 'fileUpload' in file:
-            self.handle_uploaded_file(file['fileUpload'], course_id)
-            return Response(status=status.HTTP_204_NO_CONTENT)
-        return Response(status=status.HTTP_404_NOT_FOUND)
+            return Response(self.handle_uploaded_file(file['fileUpload'], course_id))
+        return Response(status=status.HTTP_400_BAD_REQUEST)
 
     def handle_uploaded_file(self, file, course_id):
```

Comment:
@andrey-canon @diegomillan do we need to worry about the fact that someone can upload a file with a lot of lines (it can cause an slow response as it's a synchronous process)?

Reply:
Don't worry